pkgs/development/interpreters/evcxr/default.nix

```diff
@@ -1,21 +1,20 @@
-{ cargo, fetchFromGitHub, makeWrapper, pkgconfig, rustPlatform, stdenv, gcc, Security }:
+{ cargo, fetchFromGitHub, makeWrapper, pkgconfig, rustPlatform, stdenv, gcc, Security, zeromq }:
 
 rustPlatform.buildRustPackage rec {
   name = "evcxr-${version}";
-  version = "0.3.3";
+  version = "0.4.3";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "evcxr";
-    rev = "ae07ccf08723b7aec0de57d540822b89088ca036";
-    sha256 = "1apc93z9vvf6qks5x2pad45rnrj9kjl812rj78w5zmmizccp2fhf";
+    rev = "58ef3c7914ab0d958144fe8615a10c9653b217d0";
```

Comment:
Author just tagged all their releases https://github.com/google/evcxr/releases/tag/v0.4.3 :sparkles:

Reply:
Awesome! Thanks a lot for taking care of this! :)